pymedphys/tests/dicom/test_anonymise.py

```diff
@@ -346,6 +380,8 @@ def test_anonymise_directory(tmp_path):
 def test_anonymise_cli(tmp_path):
     test_file_path = get_rtplan_test_file_path()
     _test_anonymise_cli_for_file(tmp_path, test_file_path)
+    test_file_path = get_treatmentrecord_test_file_path()
+    _test_anonymise_cli_for_file(tmp_path, test_file_path)
```

Comment:
For the sake of reduced logic duplication, keeping the "list of filepaths to test" in the one place, and making it easier to expand to more files later, what do you think about the following:

```python
def get_test_filepaths():
    return [get_rtplan_test_file_path(), get_treatmentrecord_test_file_path()]

def test_anonymise_cli(tmp_path):
    for test_file_path in get_test_filepaths():
        _test_anonymise_cli_for_file(tmp_path, test_file_path)

    # ... etc

# ...

def test_anonymise_file():
    for test_file_path in get_test_filepaths():
        _test_anonymise_file_at_path(test_file_path)

    # ... etc
```

Reply:
Agree 100%